src/theme/base/_variables.scss

```diff
@@ -1,4 +1,10 @@
 $font-main: 'Open Sans', sans-serif;
+$fz-giant: 4.25rem;
 
 $color-main: #EB6E64;
 $color-white: #FFF;
+$color-text-heading-green: #32B88C;
```

Comment:
What do you think about generalizing the font color variable names?

Reply:
@charlesjliu What do you mean generalizing? Can you explain?

Reply:
I think @charlesjliu means that instead of using `$color-text-heading-green` declare the palette of colors that @emmawedekind defined already assigning names like `$green500` (for example).

Reply:
Here's the color palette @emmawedekind has defined.

<img width="731" alt="screen shot 2018-09-20 at 10 45 43 am" src="https://user-images.githubusercontent.com/219207/45830268-64770880-bcc2-11e8-915e-d87439cb7dbb.png">

Reply:
@crysfel thanks 👍